### ISSUE TITLE
Crash on 4.0.6-edge2

### ISSUE THREAD

**Issue by sebmartel** [AppGyver/steroids#964]
Happens every single time once app is up.  We use plugins.

```
06-01 15:10:47.210: E/AndroidRuntime(13901): FATAL EXCEPTION: IntentService[GCMIntentService-GCMIntentService-1]
06-01 15:10:47.210: E/AndroidRuntime(13901): Process: com.myneighbor.mobile.adhoc, PID: 13901
06-01 15:10:47.210: E/AndroidRuntime(13901): java.lang.NullPointerException
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at org.apache.cordova.NativeToJsMessageQueue.enqueueMessage(NativeToJsMessageQueue.java:250)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at org.apache.cordova.NativeToJsMessageQueue.addJavaScript(NativeToJsMessageQueue.java:219)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at com.appgyver.cordova.AGCordovaWebView.sendJavascript(AGCordovaWebView.java:108)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at com.plugin.gcm.PushPlugin.sendJavascript(PushPlugin.java:104)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at com.plugin.gcm.GCMIntentService.onRegistered(GCMIntentService.java:43)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at com.google.android.gcm.GCMBaseIntentService.handleRegistration(GCMBaseIntentService.java:296)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at com.google.android.gcm.GCMBaseIntentService.onHandleIntent(GCMBaseIntentService.java:197)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at android.os.Handler.dispatchMessage(Handler.java:102)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at android.os.Looper.loop(Looper.java:157)
06-01 15:10:47.210: E/AndroidRuntime(13901): 	at android.os.HandlerThread.run(HandlerThread.java:61)
06-01 15:10:47.290: D/com.appgyver.app.CordovaFacade(13901): Paused the application!
```

**Comment by harsaharsa** [comment]
Does the same crash happen with 4.0.6-edge1?

**Comment by harsaharsa** [comment]
This should now be fixed in 4.0.6-edge3!
